### ISSUE TITLE
Node with name "disease"

### ISSUE THREAD

**Issue by dkoslicki** [RTXteam/RTX#126]
On http://rtx.ncats.io:7574/browser/:
```
match (n) where n.rtx_name="DOID:4" return n
```
@saramsey, did we want this node to be in there? It's causing a minor issue with the NLP (since any time the user has "disease" in their question, this node is returned). I've fixed this by ignoring that node in the NLP, but it could potentially cause issues with the other questions as well (eg. DOID:4 has every phenotype, so it will be returned when asking "what diseases have symptoms similar to malaria" etc.).

**Comment by saramsey** [comment]
@dkoslicki  No, it is not in there by design.  It got into the KG because I imported the entire Disease Ontology set of nodes.  If it is is likely to mess up some parts of the reasoning engine, I propose that we remove it, using a two-pronged approach:  (1) you can nuke it from the DEV neo4j instance using Cypher, and (2) I will make sure it is not included in the next (new) version of the KG that I generate.  Deal?

**Comment by dkoslicki** [comment]
Sounds good! With the proviso that I'm rarely/never on the dev instance (so I may need you to send me via email the login instructions and how to get in the container again), so it may be easier for you to nuke the node.

**Comment by saramsey** [comment]
@dkoslicki You shouldn't need to log into rtx.ncats.io to delete a node.  Just go to the Neo4j web UI (http://rtx.ncats.io:7574, but make *sure* that your bolt connection is to the nonstandard port 7787 and not the default 7687) and type: something like this:

    MATCH (n:disease { id: "DOID:4" }) DELETE n

**Comment by saramsey** [comment]
If you are not entirely sure your bolt connection is to 7787 (and I'm not sure how to access that information through the Neo4j web UI once you're already connected), we can briefly shut down Neo4j on the PRODUCTION container; this will eliminate any chance of accidentally doing this on the wrong Neo4j.

**Comment by edeutsch** [comment]
In the past I've heard you say you made a certain neo4j instance "read-only". Would it be good policy to always have the neo4j instance that is playing the role of production server be read-only? That would mitigate this problem above?

**Comment by dkoslicki** [comment]
@saramsey ah, ok. I can do that (and I know how to check if the web port is 7787). And @edeutsch is right, we made the database read-only:
```
Neo.ClientError.General.ForbiddenOnReadOnlyDatabase
No write operations are allowed on this database. This is a read only Neo4j instance.
```

**Comment by saramsey** [comment]
OK, the DEV neo4j database is now read-write

**Comment by saramsey** [comment]
> In the past I've heard you say you made a certain neo4j instance "read-only". Would it be good policy to always have the neo4j instance that is playing the role of production server be read-only? That would mitigate this problem above?


@edeutsch yes, and I have just checked and verified that the PRODUCTION database is read-only

**Comment by saramsey** [comment]
@dkoslicki can you please ping me when you want me to switch the DEV neo4j db back to read-only?

**Comment by dkoslicki** [comment]
@saramsey Node deleted:
```
match (n:disease{id:"DOID:4"}) detach delete n
```

**Comment by saramsey** [comment]
OK the neo4j database in the DEV instance is read-only again (just restarted the neo4j server).

**Comment by dkoslicki** [comment]
Awesome! Feel free to close this issue after the KG construction code has been changed.
